tests/testthat/test-target_market_share.R

```diff
@@ -1175,3 +1177,135 @@ test_that("`target_market_share` outputs as expected for companies with 0
   expect_equal(out$projected$technology_share, c(0, 1))
   expect_equal(out$target_sds$technology_share, c(0, 0))
 })
+
+test_that("outputs columns `percent_change_by_scope` and `scope`", {
+  out <- target_market_share(
+    fake_matched(),
+    fake_ald(),
+    fake_scenario(),
+    region_isos_stable
+  )
+
+  expected_added_columns <- c( "scope", "percentage_of_initial_production_by_scope")
+
+  expect_equal(setdiff(expected_added_columns, names(out)),character(0))
```

Comment:
is this not the same as `expect_equal(names(out), expected_added_columns)`?

(for me, it's a bit more logical to put the thing you're testing first, and the reference you're testing against 2nd

Reply:
Not necessarily, the order of the columns could be different